sdk/servicebus/service-bus/test/streamingReceiver.spec.ts

```diff
@@ -230,6 +236,76 @@ describe("Streaming - Misc Tests", function(): void {
     );
     await testManualComplete();
   });
+
+  it("Bug fix #5548 - Correctly instantiates streaming receiver", async function(): Promise<void> {
+    class TestTokenProvider extends SasTokenProvider {
+      private firstCall = true;
+      private connectionStringObj: any;
+      constructor(connectionStringObj: any) {
+        super(
+          connectionStringObj.Endpoint,
+          connectionStringObj.SharedAccessKeyName,
+          connectionStringObj.SharedAccessKey
+        );
+        this.connectionStringObj = connectionStringObj;
+      }
+
+      async getToken(): Promise<TokenInfo> {
+        if (this.firstCall) {
+          this.firstCall = false;
+          throw new Error("test");
```

Comment:
We should have added an assertion that the error thrown gets handled by user provided error handler as expected?